app/views/ctc/questions/had_dependents/edit.html.erb

```diff
@@ -1,16 +1,27 @@
+<% puerto_rico_filing = current_intake.puerto_rico_filing? %>
 <% content_for :form_question, t("views.ctc.questions.had_dependents.title") %>
 
 <% content_for :form_card do %>
   <h1 class="form-question"><%= t("views.ctc.questions.had_dependents.title") %></h1>
 
   <p><%= t("views.ctc.questions.had_dependents.help_text") %></p>
 
-  <%= render('components/molecules/reveal', title: t("views.ctc.questions.had_dependents.which_relationships_qualify_reveal.title")) do %>
-    <ul class="list--bulleted">
-      <% t("views.ctc.questions.had_dependents.which_relationships_qualify_reveal.content").each do |item| %>
-        <li><%= item %></li>
-      <% end %>
-    </ul>
+  <% if puerto_rico_filing %>
```

Comment:
move this down to right past line 11, right before the loop, and only if else the loop